package.json

```diff
@@ -48,11 +48,74 @@
         "command": "tacosontitan.toolbox.azure.devops.createDefaultTasks",
         "title": "Hazel's Toolbox: Create Default Azure DevOps Tasks"
       },
+      {
+        "command": "tacosontitan.toolbox.azure.devops.setWorkItem",
+        "title": "Set Work Item",
+        "icon": "$(search)"
+      },
+      {
+        "command": "tacosontitan.toolbox.azure.devops.refreshTasks",
+        "title": "Refresh",
+        "icon": "$(refresh)"
+      },
+      {
+        "command": "tacosontitan.toolbox.azure.devops.addTask",
+        "title": "Add Task",
+        "icon": "$(add)"
+      },
+      {
+        "command": "tacosontitan.toolbox.azure.devops.changeTaskState",
+        "title": "Change State"
```

Comment:
Command titles should follow the convention `Hazel's Toolbox: <COMMAND_DISPLAY_NAME>`.